tests/e2e/100-build/200-base-href.ts

```diff
@@ -0,0 +1,8 @@
+import {ng} from '../utils/process';
+import {expectFileToMatch} from '../utils/fs';
+
+
+export default function() {
+  return ng('build', '--base-href', '/myUrl')
```

Comment:
Can we also test the default? I suppose it's not strictly needed but would be nice to check.